### ISSUE TITLE
Expression variable for the name of the print output file

### ISSUE THREAD

**Issue by guenterw** [3liz/lizmap-web-client#3892]
It would be nice if the name for the document of the print output could be defined via a variable.

Possibly similar to what is possible in QGIS for atlas printing.

**Comment by Gustry** [comment]
Indeed, it can be either the name of the layout or the expression defined for the atlas.

**Comment by cxcandid** [comment]
@guenterw I guess that's very easy to implement, because QGIS Server uses the additional URL parameter "FILE_NAME" to specify a file name for the HTTP response. This works with GETPRINT requests as well.

**Comment by nboisteault** [comment]
@cxcandid Did you test that `FILE_NAME` works with GetPrint with all formats? I read that it is only for GetMap requests and .dxf format. https://docs.qgis.org/3.34/fr/docs/server_manual/services/wms.html#getmap

**Comment by cxcandid** [comment]
@nboisteault I use the URL parameter `FILE_NAME` with GetPrint in production to generate PDF outputs (QGIS Desktop 3.30.1 / QGIS Server 3.5.6 Windows). Didn't test any other format.

**Comment by rldhont** [comment]
Fixed by https://github.com/3liz/lizmap-web-client/pull/6224
